README.md

```diff
@@ -449,7 +450,218 @@ Devices[https://arxiv.org/pdf/1807.11164](https://arxiv.org/pdf/1807.11164)
 - [STDC](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/segmentation_models/stdc.py) - Rethinking BiSeNet For Real-time Semantic Segmentation [https://arxiv.org/pdf/2104.13188](https://arxiv.org/pdf/2104.13188)
   
 </details>
-  
+
+## Training Recipes
+
+We defined recipes to ensure that anyone can reproduce our results in the most simple way.
+
+
+**Setup**
+
+To run recipes you first need to clone the super-gradients repository:
+```
+git clone https://github.com/Deci-AI/super-gradients
+```
+
+You then need to move to the root of the clone project (where you find "requirements.txt" and "setup.py") and install super-gradients:
+```
+pip install -e .
+```
+
+Finally, set the PYTHONPATH environment variable: (Replace "YOUR-LOCAL-PATH" with the path to the downloaded repo):
+```
+export PYTHONPATH=<YOUR-LOCAL-PATH>/super_gradients/
+```
+
+
+**How to run a recipe**
+
+The recipes are defined in .yaml format and we use the hydra library to allow you to easily customize the parameters.
+The basic basic syntax is as follow:
+```
+python src/super_gradients/examples/train_from_recipe_example/train_from_recipe.py --config-name=<CONFIG-NAME> dataset_params.data_dir=<PATH-TO-DATASET>
+```
+But in most cases you will want to train on multiple GPU using this syntax:
```

Comment:
multiple GPU's *

Reply:
changed in 39e462409ea6ad24070e3764025a1b273051e793